flang/test/Fir/external-mangling-emboxproc.fir

```diff
@@ -0,0 +1,10 @@
+// RUN: fir-opt --external-name-interop %s | FileCheck %s
+// RUN: tco --external-name-interop %s | FileCheck %s
+
+func @_QPfoo() {  
+  %e6 = fir.alloca tuple<i32,f64>
+  %0 = fir.emboxproc @_QPfoo_impl, %e6 : ((!fir.box<!fir.type<derived3{f:f32}>>) -> (), !fir.ref<tuple<i32,f64>>) -> !fir.boxproc<(!fir.box<!fir.type<derived3{f:f32}>>) -> ()>  
```

Comment:
Is it legal to refer to a function symbol not defined in a nmlir::FuncOp ?

Reply:
I think not much is enforced on `emboxproc` at the moment. This is just a simple test to check whether the name is changed.